### PR TITLE
Solving errors executing ETL tasks (embedded simplification)

### LINKED ISSUE #383: Errors executing ETL tasks
The execution of the BR task with the simplification embedded has raised some errors that we need to solve

- CA ETL task:
The name `ca.statcan.geo.importgeography_pr_0` in the `simplifications.json` is not correct (should be `ca.statcan.geo.importgeography_pr_`)

- BR ETL task:
```
ValueError: The following columns of the table "observatory.obs_d8684e697defcba33e8e987ba58a010d0bdf8a0b" are aggregated as 'median' or 'average' but lack of 'universe' target: br.data.Basico_V011, br.data.Basico_V009, br.data.Basico_V005, br.data.Basico_V003, br.data.Basico_V007
```

- EU ETL task:
```
luigi.target.FileAlreadyExists: Destination exists: tmp/simplification/eu.geo.ImportSHNGeoms__99914b932b_simpl
```

- EU ETL task:
```
ERROR: [pid 1] Worker Worker(salt=124385866, workers=1, host=273161c668b1, username=root, pid=1) failed    tasks.eu.eurostat.TableEU(table_name=EDUC_RENRLRG1, subsection=education, nuts_level=2, unit=people, year=2003)

ValueError: The following columns of the table "observatory.obs_f4fcec0d07615826962078191fab954c9d3ebc19" contain only NULL values: educ_renrlrg1_f_nr_ed3_g, educ_renrlrg1_m_nr_ed3_g, educ_renrlrg1_t_nr_ed3_g, educ_renrlrg1_f_nr_ed3_vpv, educ_renrlrg1_m_nr_ed3_vpv, educ_renrlrg1_t_nr_ed3_vpv, educ_renrlrg1_f_nr_ed4_g, educ_renrlrg1_m_nr_ed4_g, educ_renrlrg1_t_nr_ed4_g, educ_renrlrg1_f_nr_ed4_vpv, educ_renrlrg1_m_nr_ed4_vpv, educ_renrlrg1_t_nr_ed4_vpv
```

- US ETL task: 
Can't find an entry in `simplifications.json` for some tables (`congressional_district` , 
`school_district_unified`, `census_tract`, `block_group`, `school_district_secondary`, `school_district_elementary`)

### PR DESCRIPTION
**Pull request by antoniocarlon**
- Solved CA json problem
 - Solved BR averages without universe denominator problem
 - Solved EU null columns problem + `luigi.target.FileAlreadyExists`
 - Solved US no simplification found for some tables

Fixes #383

### CODE REVIEW

File: tasks/br/data.py [hunk 1]
```diff
@@ -257,7 +256,9 @@ def columns(self):
                 targets_dict = {}
                 for x in denominators:
                     x = x.strip()
-                    targets_dict[cols.get(x, column_reqs[x].get(session) if x in column_reqs else None)] = 'denominator'
+                    target_type = 'universe' if col_agg in ['median', 'average'] else 'denominator'
+                    targets_dict[cols.get(x, column_reqs[x].get(session)
```
Comment (ethervoid) [inline tasks/br/data.py]: For readability, we should change `x` for `denom` or `denominator`

Another question, what is the effect to have a target with universe?

File: tasks/eu/geo.py [hunk 2]
```diff
@@ -56,6 +56,7 @@ def output(self):
 
 
 class ImportSHNGeoms(Shp2TempTableTask):
+    level = IntParameter()
```
Comment (ethervoid) [inline tasks/eu/geo.py]: Why is this change needed?

File: tasks/simplifications.json [hunk 3]
```diff
@@ -437,70 +437,70 @@
 		"geomfield": "the_geom",
 		"factor": "8",
 		"maxmemory": "8192",
-		"skipfailures": "no"
+		"skipfailures": "yes"
```
Comment (ethervoid) [inline tasks/simplifications.json]: Why do we have to enable this?

Comment (antoniocarlon) [reply tasks/simplifications.json]: The TIGER geometries are the reason to have the `skipfailures` property in this JSON in the first place. `ogr2ogr` gives a lot of warnings but returns an error signal that makes the task fail.

File: tasks/eu/geo.py (hunk 2 above)
Comment (antoniocarlon) [reply tasks/eu/geo.py]: In EU we have three different simplifications for the three existing levels (see [here](https://github.com/CartoDB/bigmetadata/blob/08e5b5fafe65f0469bc108c83b0e34ac4d76eab9/tasks/simplifications.json#L248-L271)). We need a to identify the level with the task id

File: tasks/br/data.py (hunk 1 above)
Comment (antoniocarlon) [reply tasks/br/data.py]: universes are used to denormalize average and median aggregations (it doesn't make sense to normalize an average for example using the area of the geometry).

It's used [here](https://github.com/CartoDB/observatory-extension/blob/6066ef028dc428d409baa6e64bf2010fa0f38064/src/pg/sql/41_observatory_augmentation.sql#L708-L724) and [here](https://github.com/CartoDB/bigmetadata/issues/326) you can find an extended explanation

File: tasks/simplifications.json (hunk 3 above)
Comment (ethervoid) [reply tasks/simplifications.json]: Yep but we have it disabled and now is enable. Why didn't we have it enabled and why is required now?

Comment (antoniocarlon) [reply tasks/simplifications.json]: We manually simplified the `obs_xxx` tables for TIGER (final ETL tables) and now we are automatically simplifying the original TIGER tables
